src/sugar3/graphics/toolbutton.py

```diff
@@ -28,6 +27,11 @@
 from sugar3.graphics.icon import Icon
 from sugar3.graphics.palette import Palette, ToolInvoker
 
+'''
+Toolbutton is a simple button which can contain only
+one event, for example, buttons in toolbars.
```

Comment:
Please address @quozl's comments on this.  Also, could you find/write an example?  An example would be really helpful!